docs/.eslintrc.cjs

```diff
@@ -0,0 +1,52 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+	extends: [
+		require.resolve("@fluidframework/eslint-config-fluid/minimal-deprecated"),
```

Comment:
Any reason why we didn't go with `recommended`?

Reply:
That's a very good question... I have become what I hate... will fix!